scripts/bench_nvfp4_ce_kernel.py

```
@@ -0,0 +1,205 @@
+    dense_weight = head.weight.detach()
+    liger, liger_exc = _maybe_liger()
+    if liger_exc is not None:
+        print(f"liger=unavailable ({type(liger_exc).__name__}: {liger_exc})")
+
+    def materialized_loss(x):
+        return F.cross_entropy(
+            (x @ dense_weight.t()).float(), labels, ignore_index=-100
+        )
+
+    def old_loss(x):
+        return fused_fp4_cross_entropy(x, head, labels, shift=False, fp4_matmul=False)
```

Ruff (0.15.20)

[error] Undefined name `head`

(F821)